src/getting-started/starting-stack.md

```diff
@@ -11,38 +66,150 @@ First, ensure that your Docker client is up-to-date and that the `DOCKER_DEFAULT
 export DOCKER_DEFAULT_PLATFORM=linux/amd64
 ```
 
-**NOTE:** Additionally, if you have an Intel chip (ie, x86_64), you may encounter the following error when executing `docker compose up`; we recommend removing the `--platform=linux/arm64` flag within [Line 1: Dockerfile]:
+#### Declare your `config.toml`
```

Comment:
By running the `arch-cli init` command users will have the config.toml set up for them with defaults and they can then go in there and tweak it if they like by running `arch-cli config edit` or view it with `arch-cli config view`.

Reply:
This structure is also outdated and because I'm still working on it we will need to update this at a later date.